### DISCUSSION TITLE
Single-file system is no more - why leave it one file?

### DISCUSSION THREAD

**Discussion by LercDsgn** [corpengine/corpengine#29] [Questions] [answered]
### Single-file system is no more - why not leave it one file?
With the pip release, the "Only single file!" marketing is over. Wouldn't making the more general classes seperate files make it easier to develop, debug, update and manage?

**Comment by polaritynord** [comment, marked as answer]
You have a good point. We're actually planning to seperate colors, version, and constants modules to a seperate file, since having them on a single file isn't useful anymore after the Pypi release 1.0. Maybe I can move the objects to a different module too but service classes will stay.
